packages/audioplayers/lib/src/audio_pool.dart

```diff
@@ -0,0 +1,126 @@
+import 'dart:async';
+
+import 'package:audioplayers/audioplayers.dart';
+import 'package:flutter/foundation.dart';
+import 'package:synchronized/synchronized.dart';
+
+/// Represents a function that can stop an audio playing.
+typedef StopFunction = Future<void> Function();
+
+/// An AudioPool is a provider of AudioPlayers that are pre-loaded with an asset
+/// to minimize delays.
+///
+/// All AudioPlayers are loaded with the same file from `soundPath` or [source].
```

Comment:
```suggestion
/// All AudioPlayers are loaded with the same audio [source].
```
Since it could also be a url or sth else. Or we have to explicitly mention `createFromAsset` (?)